server.go

```diff
@@ -128,6 +142,14 @@ func (s *Server) Start() error {
 		return nil
 	}
 
+	// Now that we have lnd, lets extract the current version so we can use
+	// this to gate features that we'll use based on the functionality
+	// available.
+	s.lndVersion, err = s.lndServices.Versioner.GetVersion(context.Background())
```

Comment:
We already have this in `lndServices.Version`, no need to call it again.

Reply:
Nice, I must've missed that when initially looking over the `godoc`.